docs/02-admin/FAQ.md

```diff
@@ -107,6 +107,18 @@ so for a correct ready count you have to go to the queue specific overview.
 | --------------------------------------------------------- | --------------------------------------------------- | ------------------------------------------------------------------- |
 | ![Queued messages](../images/rabbit_queue_overview.png) | ![Queue overview](../images/rabbit_queue_tab.png) | ![Message Queue Overview](../images/rabbit_messages_overview.png) |
 
+## Discarding queued messages
+
+If you believe you have a queued message that is infinitely looping / stuck, you can discard it by setting the `Get messages` `Ack mode` in RabbitMQ to `Reject requeue false` with a `Messages` setting of `1` and clicking `Get message(s)`.
+
+:::warning
+
+This will permanently discard the payload
+
+:::
+
+![image](https://github.com/MbinOrg/mbin/assets/146029455/8865f0fe-f763-4980-bfc4-34b8f6b3b491)
```

Comment:
I don't know if this is a new image, but if yes it should be in the images folder inside of the docs folder